tests/e2e/specs/Modeler.spec.js

```diff
@@ -405,4 +405,25 @@ describe('Modeler', () => {
       .find('>text')
       .should('have.css', 'display', 'block');
   });
+
+  it('can collapse and expand inspector panel', () => {
```

Comment:
Can we break this into two separate tests? A failure here would make it harder to understand if collapsing or expanding the panels is broken...